presto-spi/src/main/java/io/prestosql/spi/connector/ConnectorMetadata.java

```diff
@@ -521,6 +522,22 @@ default void dropView(ConnectorSession session, SchemaTableName viewName)
         return Optional.empty();
     }
 
+    /**
+     * Gets the schema properties for the specified schema.
+     */
+    default Map<String, Object> getSchemaProperties(ConnectorSession session, CatalogSchemaName schemaName)
+    {
+        return Collections.emptyMap();
+    }
+
+    /**
+     * Get the schema properties for the specified schema.
+     */
+    default Optional<PrestoPrincipal> getSchemaOwner(ConnectorSession session, CatalogSchemaName schemaName)
+    {
+        return Optional.empty();
```

Comment:
I think it should throw by default.

Reply:
Same here, should `SHOW CREATE SCHEMA` fail on every connector except the Hive connector?